plugins/Marketplace/templates/plugin-list.twig

```diff
@@ -8,7 +8,7 @@
                             {% import '@Marketplace/macros.twig' as marketplaceMacro %}
                             {% import '@CorePluginsAdmin/macros.twig' as pluginsMacro %}
 
-                                <img src="{{ plugin.previewImage }}" alt="" class="preview-image">
+                                <img src="{{ plugin.coverImage }}?w=880&h=480" alt="" class="cover-image">
```

Comment:
The query params specifying expected with and height of the image from marketplace do no harm if the image is local, so no extra logic was applied here.